core/src/main/java/nl/nn/adapterframework/core/PipeLineSession.java

```diff
@@ -107,10 +109,8 @@ public PipeLineSession(@Nonnull Map<String, Object> t) {
 	 *                   If an empty string then no keys will be copied.
 	 * @param from Child {@link PipeLineSession} from which keys are copied.
 	 * @param to Parent {@link PipeLineSession} or {@link Map}.
-	 * @param requester Tag of where the request to copy comes from so this can be logged when
-	 *                  closing any messages.
 	 */
-	public static void mergeToParentSession(String keysToCopy, PipeLineSession from, Map<String,Object> to, INamedObject requester) {
+	public static void mergeToParentSession(String keysToCopy, PipeLineSession from, Map<String,Object> to) {
```

Comment:
Moet deze nog static zijn? from kan ook this zijn.

Reply:
Aangepast!